.codecov.yml

```diff
@@ -1,13 +1,38 @@
-comment: off
+# Overall settings for PR integration via codecov.io
+# See https://docs.codecov.io/docs/codecovyml-reference
+codecov:
+  bot: 'amphtml-codecov-bot'
```

Comment:
nit: I would either go `amp-coverage-bot` like our others (ex. `amp-owners-bot`, `amp-invite-bot`) or just `codecov-bot` like `renovate-bot` and (iirc) `bundle-size-bot`

Reply:
Excellent idea. Done.